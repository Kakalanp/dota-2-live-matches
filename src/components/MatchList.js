import { Link } from 'react-router-dom';

export default function Matchlist() {
    return (
        <div>
            <h1>Top Dota 2 live Matches</h1>
            <div>
                <Link to="matchinfo">
                    <h5>Match id:<strong>6522448049</strong></h5>
                </Link>
            </div>
        </div>
    )
}