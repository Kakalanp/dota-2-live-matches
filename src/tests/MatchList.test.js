import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import store from '../redux/configureStore';
import MatchList from '../components/MatchList';

it('is rendering succesfully', () => {
  render(<BrowserRouter><Provider store={store}><MatchList /></Provider></BrowserRouter>);
  expect(screen.getByText('Top Dota 2 live Matches')).toBeInTheDocument();
});

it('is equal to the snapshot', () => {
  const tree = renderer.create(
    <BrowserRouter><Provider store={store}><MatchList /></Provider></BrowserRouter>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
