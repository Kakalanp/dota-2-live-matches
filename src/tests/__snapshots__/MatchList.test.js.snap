// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`is equal to the snapshot 1`] = `
<div>
  <h1
    className="title"
  >
    Top Dota 2 live Matches
  </h1>
  <div
    className="list"
  />
</div>
`;
